src/platform/input/sdl2mouse.cpp

```diff
@@ -40,11 +41,13 @@ void SDL2Mouse::Load_Cursor(const char *file_location, uint32_t load_index, uint
     SDL_Surface *surface = SDL_LoadBMP(file_location);
     if (!surface) {
         // Failed to load surface
+        captainslog_error("Load_Cursor: Failed to load ANI cursor [%s]\n", file_location);
```

Comment:
`\n` can be omitted in all logs, because captainslog always appends it already.

Reply:
Copied this from different places. I'll remove it though